pkg/dt/fdt.go

```diff
@@ -211,15 +218,14 @@ func (fdt *FDT) readStructBlock(f io.ReadSeeker, strs []byte) error {
 			// The name is a null-terminating string.
 			for {
 				c := make([]byte, 1)
```

Comment:
I think it'd be nice if AlignReader implements ReadByte method (io.ByteReader interface)

Reply:
Done!